proto/config.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto2";
+package proto;
+
+option go_package = "github.com/googleinterns/aso_sxs_viewer/proto";
+
+message ViewerConfig {
+    optional int32 browser_window_count = 1;
+    optional CSSSelector css_selector = 2; //search bar selector
+    optional string url = 3;
+    optional bool use_cookies = 4;
+    optional string user_data_dir_prefix = 5;
+    repeated BrowserConfig window_overrides = 6;
+    enum WindowPosition {
+        BOTTOM = 0; //default
```

Comment:
0 should always be the default, but google specifies that enum value with 0 should specify this in name. So make the enum DEFAULT_BOTTOM = 0; and remove the comment.